internal/db/db.go

```diff
@@ -95,6 +95,9 @@ func Open(dbType DbType, connectionUrl string, opt ...Option) (*DB, error) {
 		wrappedOpts = append(wrappedOpts, dbw.WithLogger(opts.withGormFormatter))
 	}
 	if opts.withMaxOpenConnections > 0 {
+        if opts.withMaxOpenConnections > 0 && opts.withMaxOpenConnections < 5 {
```

Comment:
This first check is unnecessary, we already know this at this point
```suggestion
        if opts.withMaxOpenConnections < 5 {
```

Reply:
Thanks for the feedback.
Yes, my bad. If statement at line 97 which covers it.